lib/modules/manager/gradle/extract.ts

```diff
@@ -51,44 +51,37 @@ export async function extractAllPackageFiles(
       deps: [],
     };
 
-    try {
-      // TODO #7154
-      const content = (await readLocalFile(packageFile, 'utf8'))!;
-      const dir = upath.dirname(toAbsolutePath(packageFile));
+    // TODO #7154
+    const content = (await readLocalFile(packageFile, 'utf8'))!;
+    const dir = upath.dirname(toAbsolutePath(packageFile));
 
-      const updateVars = (newVars: PackageVariables): void => {
-        const oldVars = registry[dir] || {};
-        registry[dir] = { ...oldVars, ...newVars };
-      };
+    const updateVars = (newVars: PackageVariables): void => {
+      const oldVars = registry[dir] || {};
+      registry[dir] = { ...oldVars, ...newVars };
+    };
 
-      if (isPropsFile(packageFile)) {
-        const { vars, deps } = parseProps(content, packageFile);
-        updateVars(vars);
-        extractedDeps.push(...deps);
-      } else if (isTOMLFile(packageFile)) {
-        const updatesFromCatalog = parseCatalog(packageFile, content);
-        extractedDeps.push(...updatesFromCatalog);
-      } else {
-        const vars = getVars(registry, dir);
-        const {
-          deps,
-          urls,
-          vars: gradleVars,
-        } = parseGradle(content, vars, packageFile);
-        urls.forEach((url) => {
-          if (!registryUrls.includes(url)) {
-            registryUrls.push(url);
-          }
-        });
-        registry[dir] = { ...registry[dir], ...gradleVars };
-        updateVars(gradleVars);
-        extractedDeps.push(...deps);
-      }
-    } catch (err) {
```

Comment:
I guess so, yes. Removing the try-catch block became necessary with the switch to async `parseGradle`, as it could simply never throw a synchronously caught exception.

Reply:
that's wrong. if you await inside the try block, any thrown error will be catched. the await will throw the rejected value.